### PR TITLE
Split form_fields tests into different files.

### PR DESCRIPTION
**Pull request by berkerpeksag**
This pull request also changes all `self.assertEqual(foo, None)` instances to `self.assertIsNone(foo)`. I will open separate pull requests for remaining fields to make code review easier.

### CODE REVIEW

File: tests/forms_tests/field_tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from django.forms import Form
+from django.test import SimpleTestCase
+
+
+class BaseFieldTest(SimpleTestCase):
```
Comment (timgraham) [inline tests/forms_tests/field_tests/__init__.py]: I'd rather call this something like `FormFieldAssertionsMixin` and have it inherit `object`.

**Comment by timgraham** [comment]
Another thing I did as part of 3c1b572f1815c295878795b183b1957d0df2ca39 was to rephrase docstrings to make them consistent by stating the expected behavior and chop any prefix like "Test that" or "Ensure that". If you'd like to give a first pass at that too, great. Thanks for this. I think doing the remaining fields as part of this PR/commit is fine.

**Comment by berkerpeksag** [comment]
> Another thing I did as part of 3c1b572 was to rephrase docstrings to make them consistent by stating the expected behavior and chop any prefix like "Test that" or "Ensure that".

I fixed a couple of tests. I hope I didn't miss others.

> I think doing the remaining fields as part of this PR/commit is fine.

Done.

Thanks for the review.

**Comment by timgraham** [comment]
merged in 0247c9b08f8da4a2d93b9cede6c615011552b55a, thanks!

**Pull request closed without merging**